libcontainer/configs/validate/validator.go

```diff
@@ -286,3 +287,15 @@ func isHostNetNS(path string) (bool, error) {
 
 	return (st1.Dev == st2.Dev) && (st1.Ino == st2.Ino), nil
 }
+
+func ioPriority(config *configs.Config) error {
+	ioPriroty := config.IOPriority
+	if ioPriroty == nil {
```

Comment:
nit: your variable name has a typo.

nit: you can just do `if config.IOPriority == nil` instead of adding a variable which you don't use below.